code/__SPLURTCODE/DEFINES/is_helpers.dm

```diff
@@ -6,3 +6,5 @@
 #define iswendigo(A) (istype(A, /mob/living/carbon/wendigo))
 
 #define iscloudturf(A) (istype(A, /turf/open/chasm/cloud))
+
+#define isarachnid(A) (is_species(A, /datum/species/arachnid) || HAS_TRAIT(A, TRAIT_ARACHNID))
```

Comment:
So, the linter hates me because this is overwriting a previous macro define. However, it functions fine from what I can tell. Should I comment out the old macro definition?